dev/gen-anvil-state

```diff
@@ -3,29 +3,36 @@ set -euo pipefail
 
 # anvil first well-known private key
 export LOCAL_DEPLOY_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+export ANVIL_RPC_URL="127.0.0.1:8545"
 
-# Define a function to kill the anvil process
-kill_anvil() {
-    if pgrep -f anvil > /dev/null; then
-        echo "⧖ Killing existing anvil instance"
-        pkill -f anvil
-    fi
-}
+rm -f ./anvil-state-load-data.json || echo "no existing state to remove"
 
-# Step 1: Kill any existing anvil process
-kill_anvil
+mkdir -p environments
+
+rm -f ./environments/anvil.json || echo "no existing anvil environment to remove"
```

Comment:
do you use these `environments` for anything?

Reply:
`./environments` will contains the `testing.json`, `staging.json`, and `production.json` environment files when they get deployed manually, and will be included in every release.